tests/unittest_odgovlt.py

```diff
@@ -0,0 +1,57 @@
+from ckanext.harvest.harvesters.ckanharvester import CKANHarvester
+from ckanext.harvest.harvesters.ckanharvester import DatetimeEncoder
+from mock import patch
+from sqlalchemy import MetaData
+from sqlalchemy.engine.base import Engine
+from datetime import datetime
+
+ckan_object = CKANHarvester()
+datetime_object = DatetimeEncoder()
+
+
+def test_info():
+    info = ckan_object.info()
+    assert isinstance(info, dict)
+    dict_names = [
+           'name',
+           'title',
+           'description',
+           'form_config_interface']
+    for dict_name in info:
+        assert isinstance(info[dict_name], str)
+        assert isinstance(dict_name, str)
+        assert dict_name in dict_names
+    assert info['form_config_interface'] == 'Text'
+
+
+def test_connect_to_database():
+    @patch('ckanext.harvest.harvesters.ckanharvester.create_engine')
+    @patch('ckanext.harvest.harvesters.ckanharvester.config')
```

Comment:
Instead off of these decorators and nested test functions, simple use `mocker` fixture. See https://pypi.python.org/pypi/pytest-mock